app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/vulcan/data/hebe/VulcanHebeSendMessage.kt

```diff
@@ -40,10 +40,10 @@ class VulcanHebeSendMessage(
         val senderName = (profile?.accountName ?: profile?.studentNameLong)
             ?.swapFirstLastName() ?: ""
         val sender = JsonObject(
-            "Address" to senderName,
+            "Address" to data.senderAddressName,
```

Comment:
Implement checking if the address name OR hash is null - it may be when users first install this update.

If it's null, call `VulcanHebeMain(data).getStudents(data.profile, null)` or something like this.
This will request `register/hebe` again to update all the student data.